Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIScriptingCocoa.mm

```diff
@@ -361,9 +363,9 @@
 
         RefPtr extension = m_extension;
         for (NSString *scriptPath in scriptPaths) {
-            NSError *error;
-            if (!extension->resourceStringForPath(scriptPath, &error, WebExtension::CacheResult::No, WebExtension::SuppressNotFoundErrors::Yes)) {
-                recordError(error);
+            RefPtr<API::Error> error;
+            if (!extension->resourceStringForPath(scriptPath, error, WebExtension::CacheResult::No, WebExtension::SuppressNotFoundErrors::Yes)) {
+                recordError(::WebKit::wrapper(*error));
```

Comment:
Shouldn't need `::WebKit::` or `*`. It wasn't needed in the other cases in the PR.
```suggestion
                recordError(wrapper(error));
```

Reply:
Without ::WebKit::, the compiler uses WebExtensionContext::wrapper(), which is obviously wrong. This also goes for WebExtension.cpp/WebExtensionCocoa.mm, where WebExtension::wrapper() is called instead. The pointer can be removed though.

Reply:
Bummer! Might be able to do `error->wrapper()` instead. But `::WebKit::wrapper()` is fine for now. Converting `WebExtensionContext` to also use `API::Error` will eliminate this eventually.

Reply:
Yes, a lot of these cases will be eliminated when more code is ported, so I'm not terribly concerned about it. I can try error->wrapper() if that's better though